src/components/console/Console.js

```diff
@@ -1,14 +1,18 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext } from 'react';
 import styles from './styles.module.scss';
 import PropTypes from 'prop-types';
+import Button from '../base/button';
 import { consoleContext } from '../../contexts';
 const Console = ({ title }) => {
-  const { logs } = useContext(consoleContext);
-
+  const { logs, cleanLogs } = useContext(consoleContext);
+  const clearConsole = () => {
+    cleanLogs();
+  };
   return (
     <div className={styles.console}>
       <div className={styles.title}>
         console <div className={styles.subtitle}>-{title}</div>
+        <Button type="console" value="Clean console" onClick={clearConsole} />
```

Comment:
can you change the type name to another one more generic? maybe menu-button or something like that